plugins/modules/dns_delegation_info.py

```diff
@@ -56,11 +56,10 @@
         filters:
           fqdn: "delegation.example_zone."
           
-    - name: Get Delegation information by filters (e.g. fqdn,view)
+    - name: Get Delegation information by filters (e.g. fqdn)
```

Comment:
can you add a comment so we can include this later ?

Reply:
Not advisable to add such a comment in the example . A TODO has already been added in this file. 

https://github.com/infobloxopen/bloxone-ansible/blob/3256389ff81d00138bbc5da5127086d1fc0f705c/plugins/modules/dns_delegation.py#L233